src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -14,7 +14,7 @@
      {:on-press      (fn []
                        (rf/dispatch [:communities/load-category-states id])
                        (rf/dispatch [:dismiss-keyboard])
-                       (rf/dispatch [:navigate-to-nav2 :community {:community-id id}]))
+                       (rf/dispatch [:navigate-to :community-overview id]))
```

Comment:
@flexsurfer, @jo-mut updated the navigation on the communities home page so it goes directly to the new community overview page

Reply:
cc @Parveshdhull

Reply:
Hi @J-Son89,
Please use `(re-frame/dispatch [:communities/navigate-to-community {:community-id (:id community)}])`
and change `:community` to `:community-overview` in`:communities/navigate-to-community` event.

> so it goes directly to the new community overview page

Also, is the new community overview functional? I tried PR and looks like the community screen just has mocked data and channels are not opening. Is this intentional?

Reply:
thanks @Parveshdhull, it's not functional just yet. It is mostly using mock data. However I think it should be  it's a small adjustment to get it working.

For now, I will revert this change in this p.r and look to make this and other necessary changes in a follow up pr